src/kps/business/RouteCalculator.java

```diff
@@ -10,12 +10,13 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class RouteCalculator {
     private List<Route> routes;
 
     public RouteCalculator(Simulation simulation) {
-        this.routes = simulation.getRoutes();
+        this.routes = simulation.getUniqueRoutes().stream().filter(r -> !r.isDiscontinued()).collect(Collectors.toList());
```

Comment:
Just to add on to my last comment, the Kelburn Airways route is discontinued but the Qantas route is still valid.